docs/guides/python/ibis.md

```diff
@@ -3,147 +3,706 @@ layout: docu
-    .sort_by('type_size'))
-
-print(duckdb_types_table['type_name','type_category', 'type_size']
-    .filter(duckdb_types_table['type_category'] == 'NUMERIC')
-    .sort_by('type_size').compile())
-```
-
-| type_name | type_category | type_size |
-|:---|:---|:---|
-| DECIMAL   | NUMERIC       | NaN       |
-| TINYINT   | NUMERIC       | 1         |
-| UTINYINT  | NUMERIC       | 1         |
-| SMALLINT  | NUMERIC       | 2         |
-| USMALLINT | NUMERIC       | 2         |
-| INTEGER   | NUMERIC       | 4         |
-| FLOAT     | NUMERIC       | 4         |
-| UINTEGER  | NUMERIC       | 4         |
-| BIGINT    | NUMERIC       | 8         |
-| DOUBLE    | NUMERIC       | 8         |
-| UBIGINT   | NUMERIC       | 8         |
-| HUGEINT   | NUMERIC       | 16        |
-
-```sql
-SELECT t0.type_name, t0.type_category, t0.type_size 
-FROM (SELECT t1.type_name AS type_name, t1.type_category AS type_category, t1.type_size AS type_size 
-FROM duckdb_types AS t1 
-WHERE t1.type_category = CAST(? AS TEXT)) AS t0 ORDER BY t0.type_size
-```
-
-## Combining SQL and Ibis Expressions
-
-Ibis can also be used to combine SQL and relational operators. SQL can precede or follow Ibis relational operations. 
-
-```python
-print(duckdb_types_table.sql("""
-    SELECT 
-        *,
-        dense_rank() over (order by type_size) as size_rank 
-    FROM duckdb_types""")
-    .group_by('type_category')   
-    .aggregate(avg_size_rank=lambda t:t.size_rank.mean())
-
-print(duckdb_types_table.sql("""
-    SELECT 
-        *,
-        dense_rank() over (order by type_size) as size_rank 
-    FROM duckdb_types""")
-    .group_by('type_category')   
-    .aggregate(avg_size_rank=lambda t:t.size_rank.mean()).compile())
-```  
-
-| type_category |   avg_size_rank    |
-|:---|:---|
-| NUMERIC       | 4.583333333333333  |
-| COMPOSITE     | 3.6666666666666665 |
-| BOOLEAN       | 3.0                |
-| DATETIME      | 6.0                |
-| STRING        | 7.0                |
-| NaN           | 7.0                |
-
-```sql
-WITH _ibis_view_11 AS 
-(
-    SELECT 
-        *,
-        dense_rank() over (order by type_size) as size_rank 
-    FROM duckdb_types)
- SELECT t0.type_category, avg(t0.size_rank) AS avg_size_rank 
-FROM _ibis_view_11 AS t0 GROUP BY t0.type_category
-```
-
-To learn more about Ibis, feel free to continue with the [Ibis introductory tutorial](https://nbviewer.org/github/ibis-project/ibis-examples/blob/main/tutorial/02-Aggregates-Joins.ipynb)!
+DatabaseTable: penguins
+  species           string
+  island            string
+  bill_length_mm    float64
+  bill_depth_mm     float64
+  flipper_length_mm int64
+  body_mass_g       int64
+  sex               string
+  year              int64
+```
+
+Ibis is lazily evaluated, so instead of seeing the data, we see the schema of the table, instead. To peek at the data, we can call `head` and then `to_pandas` to get the first few rows of the table as a pandas DataFrame.
```

Comment:
```suggestion
Ibis is lazily evaluated, so instead of seeing the data, we see the schema of the table. To peek at the data, we can call `head` and then `to_pandas` to get the first few rows of the table as a pandas DataFrame.
```